mailgun.js

```diff
@@ -85,6 +85,18 @@ Mailgun.prototype.sendText = function(sender, recipients, subject, text) {
   var options = {};
   var callback = null;
 
+  if (typeof sender === 'undefined')
+    throw new Error('sender is undefined');
+
+  if (typeof sender !== 'string')
+    throw new Error('sender is not a string');
+
+  if (typeof recipients === 'undefined')
+    throw new Error('recipients is undefined');
+
+  if (typeof recipients !== 'string' && typeof recipients !== 'array')
```

Comment:
`typeof recipients !== 'array'` isn't going to work, as `typeof` an array is `'object'`.  One alternative would be `&& !(recipients instanceof Array)`.  Another alternative would just be to check for `recipients.length !== undefined`.

Either of those, or some other solution will do.  Whichever you prefer :)